ticker/android/app/src/main/java/dev/corebuilds/line/OverlayService.kt

```
@@ -0,0 +1,184 @@
+            setBackgroundColor(Color.TRANSPARENT)
+            isVerticalScrollBarEnabled = false
+            isHorizontalScrollBarEnabled = false
+            webViewClient = LineWebClient(this@OverlayService)
+            webChromeClient = WebChromeClient()
+        }
+        configure(wv.settings)
+        wv.loadUrl("https://${LineWebClient.HOST}/index.html?native=1&overlay=1")
+
+        try {
+            wm.addView(wv, params)
+        } catch (err: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)